counting_prawns.10s.js

```diff
@@ -116,9 +120,12 @@ const getStyledTitle = numberOfPr => `${getEmoji(numberOfPr)} ${numberOfPr}`;
 const getRepoName = prs => prs[0].base.repo.name;
 
 /* TODO: This method should have been called as soon as i get the data, not here. Needs refacor */
-const getCleanPrAssignedToYouDividedByRepos = prAssignedToYouDividedByRepos => (
-  prAssignedToYouDividedByRepos.filter(prAssignedToYou => prAssignedToYou.length)
-);
+const getCleanPrAssignedToYouDividedByRepos = prAssignedToYouDividedByRepos => {
+  if(prAssignedToYouDividedByRepos instanceof Array) {
```

Comment:
Why you feel the need of perform this check? How is it even possible that we pass an Object into it?